tests/conftest.py

```diff
@@ -118,6 +118,20 @@ def freeze_authority() -> Keypair:
     return Keypair.from_seed(bytes([6] * PublicKey.LENGTH))
 
 
+@pytest.fixture(scope="session")
+def unit_test_http_client() -> Client:
+    """Client to be used in unit tests."""
+    client = Client(commitment=Processed)
+    return client
+
+
+@pytest.fixture(scope="session")
+def unit_test_http_client_async() -> AsyncClient:
+    """Async client to be used in unit tests."""
+    client = AsyncClient(commitment=Processed)
+    return client
+
+
```

Comment:
Is it possible to use 
https://github.com/michaelhly/solana-py/pull/152/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R15-R20

instead?

Reply:
I do not see this used anywhere yet, little confused. Are you suggesting I combine these proposed unit test fixtures into one, exposing the named tuple around a sync, async, and event loop?

Reply:
@broper2 sorry. I misread your test. Please move it to integration tests. Unit tests should not be making rpc requests.

Reply:
@michaelhly in the tests, I am mocking the request call. It is not dependent on any other code or service outside solana-py library, which is why I put it as a unit tests. 

I can still move to integration tests if you prefer this, but I wanted to point out these tests have mocking for request before doing so.